clientlibs/java/pom.xml

```diff
@@ -9,9 +9,7 @@
 		at the same time that happen to rev to the same new version will be caught 
 		by a merge conflict. -->
 
-  <!-- 5.0.12 -> 5.0.13: no retries for 4xx, retry other errors once instead of 3 times -->
-
-	<version>5.0.13</version>
+	<version>5.0.14</version>
```

Comment:
I think it's worthwhile to keep the comments, it guards against two different PRs trying to claim the same version at the same time.